packages/babel-helpers/src/helpers/usingCtx.ts

```diff
@@ -39,6 +39,8 @@ export default function _usingCtx() {
         throw new TypeError(`Property [Symbol.dispose] is not a function.`);
       }
       stack.push({ v: value, d: dispose, a: isAwait });
+    } else if (isAwait) {
+      stack.push({ a: isAwait });
```

Comment:
Can you explicitly set `d` to `undefined`, given that we read it later?

Reply:
Or set it to false to save a few bytes?

Reply:
We could probably use `value` here too.

Reply:
Anything works for me as long as it's _something_ :)